index.bs

```diff
@@ -141,7 +141,7 @@ Presentation API Requirements {#requirements-presentation-api}
     {{PresentationConnection}} to an existing presentation on the
     receiver, given its [=presentation request URL=] and [=presentation ID=].
 
-6.  It must be possible to to close a {{PresentationConnection}} between a
```

Comment:
About all these typos you found and fixed: could you make a separate PR that fixes those?  It would be nice to have them fixed independently of this PR.

Reply:
https://github.com/webscreens/openscreenprotocol/pull/151